yb-voyager/src/dbzm/status.go

```diff
@@ -0,0 +1,49 @@
+package dbzm
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+const (
+	MODE_SNAPSHOT  = "SNAPSHOT"
+	MODE_STREAMING = "STREAMING"
+)
+
+type TableExportStatus struct {
+	DatabaseName string `json:"database_name"`
+	SchemaName   string `json:"schema_name"`
+	TableName    string `json:"table_name"`
+
+	FileName         string `json:"file_name"`
+	ExportedRowCount int64  `json:"exported_row_count"`
```

Comment:
Let's also include "approx table row count" in the status.

Reply:
We will need to include that in `export_status.json` file first.